content/en/users/tutorials/accessing-vm-with-ssh/_index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "Accessing virtual machines with SSH"
+linkTitle: "Access a VM with SSH"
+type: docs
+weight: 20
+description: "Accessing virtual machines with SSH"
+---
+
+## Overview
+
+This page provides an introduction of connecting from a local computer to a
+cloud host via SSH. It provides general guidelines, SSH options, tips, and
+examples for setting up the OpenStack environment.
+
+## SSH Keys
+
+The recommended method to access a cloud virtual machine is via ssh using **SSH
+keys**, you may inject your **public key** into the virtual machine, at
+deployment time, and use your **private key** to connect via ssh without any
+password.
+
+{{% alert title="Tip" color="info" %}} If you are using ssh keys in GitHub your
+public keys are available at: `https://github.com/${github_username}.keys`.
+i.e.: `wget https://github.com/github_username.keys` {{% /alert %}}
+
+### SSH username
+
+The username to use to connect to a virtual machine is dependent on the virtual
+machine image and is generally different in each operating system image:
+
+- For images available in the
+ [EGI AppDB Cloud Marketplace](https://appdb.egi.eu/browse/cloud), you should
+ be able to find the username in the image description.
+- For `official OS virtual machine images` you can use the general OpenStack
+  reference documentation on
+  [obtaining images](https://docs.openstack.org/image-guide/obtain-images.html).
+- For custom virtual machine images you need to refer to your virtual machine image
+  provider (i.e. it could be something specific like `cloudadm`).
```

Comment:
```suggestion
  provider (i.e. it could be something specific like `cloudadm`).
- For virtual machines deployed with [Infrastructure Manager](../../compute/orchestration/im/)
  the default username is `cloudadm`.
```